packages/snaps-cli/src/commands/manifest/manifest.ts

```diff
@@ -0,0 +1,72 @@
+import { isFile } from '@metamask/snaps-utils';
+
+import { ProcessedConfig } from '../../config';
+import { CommandError } from '../../errors';
+import { executeSteps, Steps } from '../../utils';
+import { manifest } from './implementation';
+
+type ManifestOptions = {
+  fix?: boolean;
+};
+
+type ManifestContext = {
+  input: string;
+  config: ProcessedConfig;
+  options: ManifestOptions;
+};
+
+const steps: Steps<ManifestContext> = [
+  {
+    name: 'Checking the input file.',
+    task: async ({ input }) => {
+      if (!(await isFile(input))) {
+        throw new CommandError(
+          `Manifest file not found: "${input}". Make sure that the \`snap.manifest.json\` file exists.`,
+        );
+      }
+    },
+  },
+  {
+    name: 'Validating the snap manifest.',
+    task: async ({ input, config, options, spinner }) => {
+      const write = getWriteManifest(config, options);
+      await manifest(input, write, spinner);
+
+      spinner.succeed('The snap manifest file is valid.');
```

Comment:
Should we print something if the manifest was updated?

Reply:
We could. Do you think a simple "The snap manifest file was updated." is enough?

Reply:
Yeah 👍